Library/Homebrew/tab.rb

```diff
@@ -133,6 +135,7 @@ def self.empty
       "built_as_bottle" => false,
       "poured_from_bottle" => false,
       "time" => nil,
+      "source_modified_time" => 0,
```

Comment:
I would like to make 0 as a valid date. Because I want `Tab#source_modified_time` always return valid date. Handling valid date and nil at the same time can be a little pain.